scripts/run-integration-tests.sh

```diff
@@ -43,6 +43,7 @@ CHARLIE_RPC_PORT=5002
 start-bootnode() {
 	local flags=(
 		"--rpc-port=${BOOTNODE_RPC_PORT}"
+		"--bootnodes="
```

Comment:
This was the needed change that was holding up the pull request. The integration test bootnode was communicating with our production bootnodes. (Remember that bootnodes no longer have the concept of an environment.) We are setting the libp2p IP below to localhost which wouldn't be able to connect to the public bootnodes, and I don't believe the connections were happening with the older libp2p versions.